src/main/scala/tilelink/Parameters.scala

```diff
@@ -534,20 +537,53 @@ class TLSlavePortParameters private(
   // Does this Port manage this ID/address?
   def containsSafe(address: UInt) = findSafe(address).reduce(_ || _)
 
+  // all we need to change about this is adding 2 members, one for the tlmaster parametrs and one for the tlslave parameters
+  // All wee need to do is change the membership function to instead of only slave pamaters it looks at the intersection of both master and slave
+  // do we need to pass in two different arguments
```

Comment:
I think we concluded below that two different helper functions are necessary and that we will `&&` the results of the duals together in the edge function. So this one should remain focused on `TLSlaveParameters => TransferSizes` cases that look at `address`.